src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/main.scss"
```

Comment:
The `href` attribute is linking to a `.scss` file. SCSS files need to be compiled into CSS before they can be linked in an HTML file. Please ensure that you link to the compiled `.css` file instead.